processor/span/config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package span
```

Comment:
We are inconsistent on the naming of these packages: should they have the processor suffix or not? Could you please open an issue so we can have a quick discussion about it and then refactor in a separate PR.

Reply:
I'll update this pr to match the changes made by Yang.